.pre-commit-config.yaml

```
@@ -0,0 +1,12 @@
+# See https://pre-commit.com for more information
+# See https://pre-commit.com/hooks.html for more hooks
+repos:
+  - repo: https://github.com/pre-commit/mirrors-autopep8
+    rev: 'master'
```

```suggestion
    rev: 'v1.5.7'
```

So that new users do not have to run `pre-commit autoupdate`?